src/store/modules/features.store.js

```diff
@@ -615,9 +623,19 @@ export default {
                 commit('setProfileFeature', { feature: feature, dispatcher })
                 commit('setProfileData', { data: null, dispatcher })
                 if (feature?.geometry) {
-                    let coordinates = [...feature.geometry.coordinates]
+                    let coordinates
+                    if (simplifyGeometry) {
+                        const turfGeom = transformIntoTurfEquivalent(feature.geometry)
+                        coordinates = [
+                            ...simplify(turfGeom, {
+                                tolerance: GPX_GEOMETRY_SIMPLIFICATION_TOLERANCE,
+                            }).geometry.coordinates,
+                        ]
+                    } else {
```

Comment:
I though it would be easier to use this one for the code's complexity, as I receive the geometry in a GeoJSON form.
I could re-parse all the features as OpenLayers features, I thought it might be a bit overkill (and preferred to depend on something else than OL in the store).
But that is totally debatable.

Reply:
both are using a Ramer-Douglas-Peucker algorithm

Reply:
Ok then fine, but would be a very good idea to document this about the algo in the git commit message as well as on the PR description, so that we know.